packages/teko/tests/src/tLSCStabilized_tpetra.cpp

```diff
@@ -555,12 +555,14 @@ bool tLSCStabilized_tpetra::test_strategy(int verbosity,std::ostream & os)
       // test alpha*inv(D)
       ss.str("");
       // result = tester.compare( *aiD, *iStrat.getInvAlphaD(blkA,state), &fos );
+      /*
```

Comment:
@eric-c-cyr, it looks like this will disable this particular test for all builds.  Is doing that going to weaken the testing of Teko?  Do you plan to try to fix this one unit test case in the short term or leave it disabled long-term?

Another option is to add a Cmake cache var and a simple define to disable just this one unit test case.  Then we can disable this one test case for just this one set of RHEL6 GCC 6.1.0 builds.   That would be a better option if this is going to be disabled long-term.

In general, we always want to disable things with the finest cuts we can make so as not to loose any existing passing tests.

Reply:
@eric-c-cyr,

BTW, I can add the CMake var and the define to make this disable more targeted, if that is okay with you?  I can even update the branch in this PR to do that.

Note that PR testing is currently down (see https://github.com/trilinos/Trilinos/pull/2651#issuecomment-385014648) so we have time to do this.

Reply:
Yes, this is fine with me. 

It will be interesting to see if this test fails on other platforms. Its essentially a copy of the Epetra test which passes (so far) on all platforms.